### PR TITLE
Add bluetooth audio packages to gateway profile

### PR DESCRIPTION
**Pull request by juimonen**
Add pulseaudio and its bluetooth related modules
to gateway profile. Includes also a simple test to
check the existence of the daemon.

Signed-off-by: Jaska Uimonen <jaska.uimonen@intel.com>

### CODE REVIEW

File: meta-iotqa/lib/oeqa/runtime/sanity/pulseaudio.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from oeqa.oetest import oeRuntimeTest
+from oeqa.utils.decorators import tag
+
+class PulseaudioDaemonTest(oeRuntimeTest):
+
+    '''Pulseaudio daemon check'''
+    def test_pulseaudio_daemoncheck(self):
+        '''check pulseaudio daemon'''
```
Comment (skuusela) [inline meta-iotqa/lib/oeqa/runtime/sanity/pulseaudio.py]: Maybe rename the the class as 'PulseaudioTest' and the test function as 'test_pulseaudio_exists' and also the comments to reflect those.

Comment (juimonen) [inline meta-iotqa/lib/oeqa/runtime/sanity/pulseaudio.py]: ok done.

**Comment by mythi** [comment]
the oe-core update is not needed. Also, for better granularity this needs three commits: add the feature, add the tests, enable the feature in gateway

**Comment by juimonen** [comment]
ok. tried to do the commit split.

**Pull request merged**